internal/logger/component.go

```diff
@@ -199,3 +211,62 @@ func SerializeConnection(conn Connection, extraKeysAndValues ...interface{}) []i
 
 	return keysAndValues
 }
+
+// Server contains data that all server messages MAY contain.
+type Server struct {
```

Comment:
Even though both "Server" and "Topology" fall under the "ComponentTopology" component, it feels more natural to separate them into two separate message structs since they will ultimately contain non-intersecting fields.